### PR TITLE
Importing External Files using Webpack Loaders: The Beginners Guide

### LINKED ISSUE #3629: Importing External Files using Webpack Loaders
### Introduction paragraph 
Using Webpack for the first time can be quite demanding. You may have external files, that you need to import and may find it difficult to import these files using webpack. 

In this tutorial, I will be giving a detailed step-by-step approach to import various files using webpack loaders. 

### Key takeaways: 
By the end of this tutorial readers should be able to:
1. Install Webpack
2. Learn how to interact with Webpack using CLI
3. Make use of Webpack loaders to import various types of files. 

### References: 
The best reference is the official [Webpack Documentation] (https://webpack.js.org/concepts/)

### Templates to use as guides
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Favour-dgreat**
Created the Tutorial article Importing External Files Using Webpack Loaders: The Beginners Guide including the images used in the article in the images folder and finally the author details in an index.md file in the authors' folder and the avatar image. 

Importing External Files using Webpack loaders is a tutorial article that addresses how to use Webpack and Webpack loaders for total beginners, so the contents in this tutorial are very concise and other supporting learning docs have been linked in the article, to further the learning process. 

This fixes #3629

### CODE REVIEW

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
+# Importing External Files using Webpack Loaders: The Beginners Guide 
+Today, I am going to show you how to use Webpack loaders and import external files while making use of Webpack. In this tutorial, I am going to be very concise, because I have understood that making use of Webpack loaders to import external files in a web project, can be quite demanding and may look like rocket science, especially for beginners, but it is actually quite easy. 
+
+## Table Of Contents
```
Comment (srishilesh) [inline content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md:4]: Start with `###`

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+1. What is Webpack?
+2. Installing Webpack
+3. Loaders and Plugins
+4. Make use of Webpack loaders to import various types of files
+
+## Prerequisites
```
Comment (srishilesh) [inline content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md:10]: `###`

Comment (srishilesh) [reply content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md:10]: Make the subsequent changes

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+## What is Webpack?
+Webpack is a tool that allows you to modularise your code and helps you to push your codes to one file in other for you not to worry about the hierarchy of your files, especially when working with JavaScript files and also helps you to inject external files, like an SVG, images, your CSS files or other modern JS codes, in your web app. Webpack provides a configuration file, where you declare how things should be sorted out and then you relax. 
+
+**Remember I stated earlier that this tutorial is going to be concise, hence a very simple definition of Webpack, which is what I have stated above, for a more elaborate definition of Webpack, check out this Wiki Definition of Webpack [Wikipedia's Definition of Webpack](https://en.wikipedia.org/wiki/Webpack)**
+
+## Installing Webpack
+The first thing you are going to do is to create a directory or folder, you can call it Webpack or anything you feel appropriate, and then you create another folder and name it Webpack Tutorial. 
+
+<img src="./images/webpack1.png" alt="File directory image" border="0"></a>
```
Comment (srishilesh) [inline content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md:23]: This is not markdown.

I would suggest you to have a look at the guidelines

**Review by srishilesh — CHANGES_REQUESTED** [review]
@Favour-dgreat 

Welcome to EngEd!

Congratulations on your first article.

On a quick glance, I noticed a few extra files like `package.json` and `package-lock.json`.

The images shouldn't be under the directory `[ARTICLE-NAME]/images`, instead it should be directly under `[ARTICLE-NAME]`

For example, `content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/images/webpack11.png` -> `content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/webpack11.png`

And, the content doesn't seem to follow the guidelines. I would suggest you to take a look at few of my articles [here](https://www.section.io/engineering-education/authors/srishilesh-p-s/), to understand the format and flow.

Ensure to add a hero image of dimensions 600x400.

Perform a grammar check using Grammarly.

And finally, don't remove the PR checklist.

Please ping me when the changes are done.

Thanks.

**Comment by ahmadmardeni1** [comment]
plagiarism check done ✅

**Comment by srishilesh** [comment]
@Favour-dgreat 

Are the changes done?

Looks like there are way too many commits. From the next time, it is advised that you group all the major changes, and commit them together.

It's not a good practice to commit for every minor change. It makes debugging and reverting changes even harder.

**Comment by Favour-dgreat** [comment]
Hi. I am quite new and I just kept on pushing new changes. I am very sorry
for that. I am done with the changes. What is next?

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md (hunk 1 above)
Comment (Favour-dgreat) [reply content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md]: Fixed

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md (hunk 2 above)
Comment (Favour-dgreat) [reply content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md]: Done

File: content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md (hunk 3 above)
Comment (Favour-dgreat) [reply content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md]: Fixed. Thank You

**Comment by srishilesh** [comment]
@Favour-dgreat  Thanks for the changes. Will start my next round of review ASAP

**Comment by srishilesh** [comment]
I'll complete my next round of review by tomorrow

**Comment by srishilesh** [comment]
@Favour-dgreat 

I've been trying to push my changes to this branch. For some reason, it's not allowing me to push the edits.

I would suggest you to create a new PR and tag me there. And, make sure you use a different branch name other than `master`.

And, just make them as a single commit.

Also, make sure to add back the `package.json` file that you created.

Ping me in the new PR. Closing this

Thanks.

**Pull request closed without merging**